Mathlib/MeasureTheory/Integral/Pi.lean

```diff
@@ -12,38 +12,49 @@ import Mathlib.MeasureTheory.Constructions.Prod.Integral
 
 open BigOperators Fintype MeasureTheory MeasureTheory.Measure
 
-theorem MeasureTheory.integral_finset_prod_eq_prod' {E : Type*} {n : ℕ} (f : (Fin n) → E → ℝ)
-    [MeasureSpace E] [SigmaFinite (volume : Measure E)] :
-    ∫ x : (Fin n) → E, ∏ i, f i (x i) = ∏ i, ∫ x, f i x := by
+variable {L : Type*} [IsROrC L]
```

Comment:
I think we try to use `\bbk` wherever possible for arguments to `IsROrC`, though perhaps that's not as established a convention as I think it is.